### PR TITLE
Translate Grammer section

### PR DESCRIPTION
**Pull request by tarvos21**
Translate Grammer section: content, spelling, syntax

Also: fix a typo in Page layout subsection

### CODE REVIEW

File: guide-en.org [hunk 1]
```diff
@@ -311,36 +311,36 @@ Limited to our background, we summarize some common spelling errors in the IT do
...
-Verbalized | Verified | Vitalized | Yielded |
+| Accelerated   | Accomplished | Accounted    | Accumulated  | Achieved     |
+| Administrated | Arbitrated   | Articulated  | Boosted      | Briefed      |
+| Broadened     | Budgeted     | Campaigned   | Chaired      | Championed   |
+| Clarified     | Coached      | Collaborated | Coordinated  | Corroborated |
+| Cultivated    | Customized   | Decided      | Decreased    | Delegated    |
+| Demonstrated  | Designated   | Developed    | Devised      | Diagnosed    |
+| Documented    | Doubled      | Economized   | Edited       | Educated     |
+| Empowered     | Enabled      | Encouraged   | Endorsed     | Enhanced     |
+| Facilitated   | Focused      | Forecast     | Generated    | Harmonized   |
```
Comment (xiaohanyu) [inline guide-en.org]: Oh, I just fixed the "Faciliated" typo in the Chinese "guide.org".

For "Forecast", I've checked the dictionary, and found that both "forecast" and "forecasted" are legal past tense for the word "Forecast". So for consistency, I think it's better to keep "Forecasted" here?

File: guide-en.org [hunk 2]
```diff
@@ -270,44 +270,44 @@ Prohibited contents:
 
 ** Spelling
 
-A spelling mistake in a resume is a job search. Too many primary spelling mistakes can affect employers' first impressions of job seekers. Most software has a spell check function, appropriate to avoid a lot of primary spelling errors. Of course, do not over-reliance on the software's spell check function, it is best to find a trusted friend for the second proof-reading.
+Spelling mistakes in a resume is a big taboo when applying for jobs. Too many primary spelling mistakes will make bad first impressions of applicants on employers. Most software have the spelling check function, using it appropriately can avoid many primary spelling errors. Of course, you can never rely too much on the software's spelling check, it's better to find a trusted friend for double proof-reading.
```
Comment (xiaohanyu) [inline guide-en.org]: Wondering if "double proof-reading" a good choice over just "proof-reading" cause "proof-reading" itself contains the meaning of "double check".

Just an advice.

Comment (tarvos21) [inline guide-en.org]: OK, yes, so proof-reading  would be enough. I just want to emphasize it, haha.

File: guide-en.org (hunk 1 above)
Comment (tarvos21) [inline guide-en.org]: Yes, both are usable, but it's quite more common or natural to use forecast. Consistency is good, but using 'Forecasted' may put consistency over usability(popularity), so how do you decide? It's on you.

File: guide-en.org (hunk 2 above)
Comment (xiaohanyu) [inline guide-en.org]: Yeah, I've also googled "double proof-reading", and found nothing. So I think here "proof-reading" is enough.

File: guide-en.org (hunk 1 above)
Comment (xiaohanyu) [inline guide-en.org]: I still prefer "forecasted" as all other words in this table end with "ed". Though both "forecasted" and "forecast" are legal past tense for "forecast", "forecasted" is more explicit, I think. You know, Python told us that "explicit is better than implicit", 😀

Comment (tarvos21) [inline guide-en.org]: Ok, so I will change it, haha

**Comment by tarvos21** [comment]
2 modification done.

**Pull request merged**